py/desispec/tile_qa.py

```diff
@@ -0,0 +1,221 @@
+"""
+desispec.exposure_qa
+============
+
+Utility functions to compute an exposure QA scores.
+"""
+
+import os,sys
+import numpy as np
+from astropy.table import Table,vstack
+import fitsio
+import yaml
+from pkg_resources import resource_filename
+import glob
+
+from desiutil.log import get_logger
+
+from desispec.exposure_qa import compute_exposure_qa,get_qa_params
+from desispec.io import read_fibermap,findfile,read_exposure_qa,write_exposure_qa
+from desispec.maskbits import fibermask
+
+
+def mystack(tables) :
```

Comment:
What's the processing case where this was needed?  Let's make this a `_mystack` hidden function and document what it does that is different from normal vstack.

Reply:
replaced by vstack after preparing the tables (rm some meta keywords, rm COEFF column for zbest).